scripts/move_to_scala_2.11.sh

```diff
@@ -2,7 +2,7 @@
 
 set +x
 
-find . -name "pom.xml" -exec sed -e "s/2.10.6/2.11.8/g" -e "s/2.10/2.11/g" -i.2.11.bak '{}' \;
+find . -name "pom.xml" -exec sed -e "s/2.10.6/2.11.8/g" -e "/bdg-utils.version/!s/2.10/2.11/g" -i.2.11.bak '{}' \;
```

Comment:
The utils version bumps to 0.2.10 in this PR. If you don't filter out that line when running the sed replacements, then it changes the utils version to 0.2.11. This doesn't happen going 2.11->2.10, since the replacement is different.